modal/_container_entrypoint.py

```diff
@@ -587,10 +646,14 @@ def main(container_args: api_pb2.ContainerArguments, client: Client):
             imp_fun.fun = run_in_pty(imp_fun.fun, input_stream_blocking, pty_info)
 
         if not imp_fun.is_async:
-            call_function_sync(function_io_manager, imp_fun.obj, imp_fun.fun, imp_fun.is_generator)
+            call_function_sync(
```

Comment:
maybe worth changing the signature of this function to take an `ImportedFunction`